### PR TITLE
Sudo error check

### PR DESCRIPTION
**Pull request by kishorikumar**

### CODE REVIEW

File: src/synopsys-action/utility.ts [hunk 1]
```diff
@@ -27,3 +27,13 @@ export async function cleanupTempDir(tempDir: string): Promise<void> {
 export function checkIfGithubHostedAndLinux(): boolean {
   return String(process.env['RUNNER_NAME']).includes('Hosted Agent') && (process.platform === 'linux' || process.platform === 'darwin')
 }
+
+export function formatAndGetErrorMessage(errorMessage: string): string {
+  const appErrorPart = errorMessage.match(".*'([^']*)'.*")?.at(1)
```
Comment (tithiSynopsys) [inline src/synopsys-action/utility.ts:32]: what is appErrorPart?

Comment (kishorikumar) [reply src/synopsys-action/utility.ts:32]: This is bridge path in the error message '/home/runner/synopsys-bridge/bridge'

**Review by tithiSynopsys — APPROVED** [review]

File: src/synopsys-action/synopsys-bridge.ts [hunk 2]
```diff
@@ -77,11 +77,7 @@ export class SynopsysBridge {
           cwd: workingDirectory
         }
         try {
-          if (checkIfGithubHostedAndLinux()) {
-            return await exec('sudo '.concat(this.bridgeExecutablePath.concat(' ', bridgeCommand)), [], exectOp)
-          }
-
-          return await exec(this.bridgeExecutablePath.concat(' ', bridgeCommand), [], exectOp)
+          return await exec(this.bridgeExecutablePath.concat(' ', bridgeCommand).concat(' --verbose'), [], exectOp)
```
[the remark this answers is not in the document]
Comment (kishorikumar) [reply src/synopsys-action/synopsys-bridge.ts]: Removed

**Review by Muninth — COMMENTED** [review]
H

**Comment by Muninth** [comment]
I

**Pull request merged**